src/utils/url.ts

```
@@ -161,6 +202,68 @@
+  }
+
+  const resource = config.typ === "S3Config" ? config.bucket : config.container;
+  const errResponse = createXmlErrorResponse(err, 502, resource);
+  throw new HTTPException(errResponse.status, {
+    res: errResponse,
+  });
+}
+
+// Function to handle exponential backoff retries
+export async function retryWithExponentialBackoffReplica<T>(
+  s3Fn: (config: S3Config) => Promise<T> | void,
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)